### PR TITLE
Navbar (& a bit of footer improvements)

### PR DESCRIPTION
**Pull request by janiseliang**
### Visit site here:
https://ucbpfa-pantry-webapp-git-navbar-cal-nourish.vercel.app/

completing issue #47 
* improvement on appearance of user profile when logged in (on the top right). also moved admin and signout links to be under dropdown below the user info
* fixed footer to be always at the bottom of the screen when content is smaller than page
* added handling for mobile/smaller screens with drop-down menu

**regular nav bar/footer:**
regular nav bar

**small screen**
for small screens

### CODE REVIEW

**Comment by janiseliang** [comment]
oh dear the 2nd screenshot is gigantic sorry... i'm not sure how to fix that

File: components/Navbar.js [hunk 1]
```diff
@@ -50,31 +54,71 @@ export default function Navbar() {
...
+            </div>
+          </div>
+        }
+      </div>
+
+      {/* Tabs */}
+      {/* drop-down menu button for small screens */}
+      <button type="button" onClick={toggleShowTabs}
+        className="inline-flex items-center p-1 ml-3 text-sm text-gray-500 rounded-lg md:hidden hover:bg-gray-100 focus:outline-none" aria-controls="mobile-menu-2" aria-expanded="false">
+        <svg className="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z" clipRule="evenodd"></path></svg>
```
Comment (samuelwshen) [inline components/Navbar.js:107]: nit: moving these SVGs out into their own file so they can be used throughout the project

**Review by samuelwshen — APPROVED** [review]
very nice! few small comments + I think there are some commented out lines of code that could be cleaned up

File: components/Navbar.js [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
+  // { title: "Admin", route: "/admin"},
+  // { title: "Sign out", route: "/signout"},
 ]
 
 
 export default function Navbar() {
-  const linkStyle = "ml-4 px-3 py-2 rounded-md text-sm font-medium leading-5 transition duration-150 ease-in-out"
-  const activeLink = `${linkStyle} text-white bg-pantry-blue-900 focus:text-white focus:bg-pantry-blue-600`
-  const inactiveLink = `${linkStyle} px-3 py-2 rounded-md text-sm font-medium text-gray-300 hover:text-white hover:bg-pantry-blue-700 focus:outline-none focus:text-white focus:bg-pantry-blue-700`
+  const linkStyle = "block py-2 pr-4 pl-3 text-white rounded hover:bg-pantry-blue-400" +
```
[the remark this answers is not in the document]
Comment (janiseliang) [reply components/Navbar.js:33]: these links could only ever be used in the navigation bar i think... i suppose a more accurate name would be navTab and navTabCurrent. they're very specifically intended for the responsive navbar

[the remark this answers is not in the document]
Comment (janiseliang) [reply components/Navbar.js:33]: could still move to a styling file though! maybe styles/index.css

**Review by jschnapper — APPROVED** [review]
These are really great changes! Clean and good design decisions. I love the user icon. I'm cool with merging this as is. One small thing: what do folks think of collapsing the sign-in button (and user icon + sign out) into the dropdown menu when the screen size is small? In any case, this is a great improvement to the layout.

From here
Screen Shot 2022-03-24 at 10 10 50 PM

To under "hours"
Screen Shot 2022-03-24 at 10 13 55 PM

**Pull request merged**